shell/src/main/java/org/apache/accumulo/shell/commands/ScanCommand.java

```diff
@@ -242,9 +248,13 @@ protected ScanInterpreter getInterpreter(final CommandLine cl, final String tabl
 
     try {
       if (cl.hasOption(formatterOpt.getOpt())) {
+        Shell.log.warn("Formatter option is deprecated and will be removed in a future version.\n");
+
         return shellState.getClassLoader(cl, shellState)
             .loadClass(cl.getOptionValue(formatterOpt.getOpt())).asSubclass(Formatter.class);
       } else if (cl.hasOption(formatterInterpeterOpt.getOpt())) {
+        Shell.log.warn("Formatter option is deprecated and will be removed in a future version.\n");
```

Comment:
Eek, this got merged with extra newlines in the log message. I'll strip them out.

Reply:
See #1154

Reply:
Doh... thanks!